en/releases/release-1.5.2.md

```diff
@@ -0,0 +1,22 @@
+---
+title: TiDB Operator 1.5.2 Release Notes
+---
+
+# TiDB Operator 1.5.2 Release Notes
+
+Release date: January 19, 2023
+
+TiDB Operator version: 1.5.2
+
+## New features
```

Comment:
From v1.5.2, TiDB operator supports back up and restore the data of a TiDB cluster deployed across multiple AWS Kubernetes clusters to AWS storage using EBS volume snapshots ([#5003](https://github.com/pingcap/tidb-operator/pull/5003), [@BornChanger](https://github.com/BornChanger), [@WangLe1321](https://github.com/WangLe1321), [@YuJuncen](https://github.com/YuJuncen), [@csuzhangxc](https://github.com/csuzhangxc)). For detailed information, visit [Back up Data Using EBS Snapshots across Multiple Kubernetes](backup-by-ebs-snapshot-across-multiple-kubernetes.md) and [Restore Data Using EBS Snapshots across Multiple Kubernetes](restore-from-ebs-snapshot-across-multiple-kubernetes.md)